tidy3d/web/core/http_management.py

```diff
@@ -9,9 +9,9 @@
 import toml
 from tidy3d.web.cli.constants import CONFIG_FILE
 
-from .environment import Env
-from ..exceptions import WebError
-from ..version import __version__
+from tidy3d.web.core.environment import Env
```

Comment:
let's use relative imports, for example `from .environment import Env`

Reply:
modified.